web/packages/extension/webpack.config.js

```diff
@@ -99,7 +100,7 @@ export default function (/** @type {Record<string, any>} */ env, _argv) {
         },
         output: {
             path: url.fileURLToPath(new URL("assets/dist/", import.meta.url)),
-            publicPath: "auto",
+            publicPath: "/dist/",
```

Comment:
This gets changed during runtime to the full URL of the extension's /dist/ folder.